develop/tutorials/articles/260-themes-and-layout-templates/07-embedding-portlets-in-themes-and-layout-templates.markdown

```diff
@@ -97,13 +106,85 @@ use wherever your theme is used.
 
 You successfully requested a portlet based on the entity and action types
 required, and created and deployed a module that retrieves the portlet and
-embeds it in your theme. Next, you'll learn a similar process to embed a portlet
-in your layout template.
+embeds it in your theme. 
+
+### Embedding a Portlet by Portlet Name [](id=embedding-a-portlet-by-portlet-name)
+
+If you'd like to embed a specific portlet in the theme, you can hardcode it by
+providing its instance ID and name:
+
+    <@liferay_portlet["runtime"]
+        portletProviderAction=portletProviderAction.VIEW
+        instanceId="INSTANCE_ID"
+        portletName="PORTLET_NAME"
+    />
+
+If your portlet is instanceable, an instance ID must be provided; otherwise, you
+can remove this line. To set your portlet to non-instanceable, set the property
+`com.liferay.portlet.instanceable` in the component annotation of your portlet
+to `false`. The portlet name must be the same as the property
+`javax.portlet.name`.
```

Comment:
> To set your portlet to non-instanceable, set the property `com.liferay.portlet.instanceable` in the component annotation of your portlet to `false`. The portlet name must be the same as the property `javax.portlet.name`.

Can this be a "side note" or something? Rather than inline.